readme.md

```diff
@@ -1,49 +1,63 @@
+ * A compiler.
+
+### Installation - yotta
+
+Pre-requisites:
 
-### Installation
-Detailed instructions can be found on the [documentation
-site](http://docs.yottabuild.org/#installing), in summary, to use yotta you
-need:
+1. yotta runs on [python](https://www.python.org/downloads/release/python-279/). Version 2.7.9 of python is recommended if you do not already have python installed. 
 
- * yotta itself
- * [CMake](http://www.cmake.org/download/)
- * a compiler
+2. To install yotta, you'll need Python's [pip](http://pip.readthedocs.org/en/latest/installing.html).
 
 **To install yotta itself**, open a terminal, then run:
+
 ``` bash
 pip install yotta
 ```
 
-yotta runs in [python](https://www.python.org/downloads/release/python-279/), and is installed using [pip](http://pip.readthedocs.org/en/latest/installing.html). Version 2.7.9 of python is recommended if you do not already have python installed. If permission is denied on Linux or OS X, you may need to run `sudo pip install yotta`. 
-
-**Install CMake** from the [CMake download page](http://www.cmake.org/download/), or using your system's package manager. Make sure to check the option in the installer to add it to your path. 
+**Tip:** If permission is denied on Linux or OS X, you may need to run ``sudo pip install yotta``. 
 
-**Which compiler** you need depends on whether you're building programs for your host system, or cross-compiling them run on an embedded device:
+### Installation - CMake
 
- * to cross-compile, install [arm-none-eabi-gcc](https://launchpad.net/gcc-arm-embedded/+download).
- * to compile natively on OS X, [install Xcode](https://developer.apple.com/xcode/downloads/), including the command-line tools.
- * to compile natively on Linux, install clang with your system's package manager.
+**Install CMake** from the [CMake download page](http://www.cmake.org/download/), or using your system's package manager. Make sure to check the option in the installer to add it to your path. 
 
-Further information on installing yotta for different platforms can be found on the [documentation site](http://docs.yottabuild.org/#installing).
+### Installation - Compilers
 
-### Get Started!
-The best way to get started is to [follow the tutorial](http://docs.yottabuild.org/tutorial/tutorial.html).
+**Which compiler** you need depends on whether you're building programs for your host system, or cross-compiling them to run on an embedded device:
 
-### What `yotta` does
-yotta downloads the software components that your program depends on (it's similar in concept to npm, pip or gem). To install a new module, you run `yotta install --save <modulename>`, and yotta will install both the module you've specified and any of its dependencies that you don't already have installed, and save the fact that you depend on that module into your module's description file.
+ * To cross-compile, install [the **ARM none-EABI GCC**](https://launchpad.net/gcc-arm-embedded/+download).
```

Comment:
arm-none-eabi-gcc is the name of the program, it should remain formatted like that